app/models/mixins/ontap_metrics_rollup_mixin.rb

```diff
@@ -70,12 +70,7 @@ def baseCounterNames(counterInfo)
 
     def find_all_by_interval_and_time_range(interval, start_time, end_time = nil, count = :all, options = {})
       my_cond = ["rollup_type = ? and statistic_time > ? and statistic_time <= ?", interval, start_time, end_time]
```

Comment:
Ok, so I hear you saying don't pass in `count` or `options` and just tack on the `find(count, options)` on in the caller.